karapace/protobuf/rpc_element.py

```diff
@@ -19,7 +21,7 @@ def __init__(
         response_type: str = "",
         request_streaming: bool = False,
         response_streaming: bool = False,
-        options: list = None,
+        options: list | None = None,
```

Comment:
question (non-blocking): Can we eliminate the implicit `Any` here, i.e provide a list item type? (I usually go about that by changing to `list[object]` to discover expectations)